x/profiles/keeper/keeper.go

```diff
@@ -32,7 +39,13 @@ type Keeper struct {
 // 2. DTag -> Address
 //    This is used to get the address of a user based on a DTag
 func NewKeeper(
-	cdc codec.BinaryMarshaler, storeKey sdk.StoreKey, paramSpace paramstypes.Subspace, ak authkeeper.AccountKeeper,
+	cdc codec.BinaryMarshaler,
+	storeKey sdk.StoreKey,
+	paramSpace paramstypes.Subspace,
+	ak authkeeper.AccountKeeper,
+	channelKeeper ibctypes.ChannelKeeper,
+	portKeeper ibctypes.PortKeeper,
+	scopedKeeper capabilitykeeper.ScopedKeeper,
```

Comment:
Instead of depending on the `ibctypes` and `capabilitykeeper` packages, I think it's better to define the expected keeper definitions inside a new file named `expected_keeper` inside the `types` folder the same way that is being done [here](https://github.com/cosmos/ibc-go/blob/main/modules/apps/transfer/types/expected_keepers.go)